src/chainstate/burn/db/sortdb.rs

```diff
@@ -3488,7 +3690,7 @@ mod tests {
         // stx:      1,    2,    3,    4,                            5,    6
         // burn:  0x01, 0x02, 0x03, 0x04, 0x45, 0x46, 0x47, 0x48, 0x29, 0x2a, 0x2b, 0x2c
 
-        last_snapshot = SortitionDB::get_canonical_burn_chain_tip_stubbed(db.conn()).unwrap();
+        last_snapshot = SortitionDB::get_canonical_burn_chain_tip(db.conn()).unwrap();
```

Comment:
There don't seem to be any new tests here for the new functionality.  Can you add some for things like:

* loading/storing burnchain operations?  Can be trivial "does it work or not"-like tests.
* parsing/serializing the PoX ID?

Reply:
loading/storing burnchain operations is tested both in the neon_integrations tests and in the `coordinator::tests` module.

I can add some tests for parsing/serializing the PoX ID though (though the PoxId implementation is very much a placeholder -- I'd expect it's serialization format to change pretty drastically when it's implemented with an actual bitvector).

Reply:
Okay, can hold off until we're happy with the PoX ID representation first.